### PR TITLE
Update Hoek and dev dependencies

### PR DESCRIPTION
**Pull request by momiller121**
dependency, maintenance

I wanted to address the hoek@5.0.4 deprecation warning. Also updated the dev dependencies.

I'd appreciate a second set of eyes on the small change I made to the [failing test on the 2.0.0 release.](https://github.com/momiller121/oppsy/commit/e6c00e4ca5d36c15c086b63d34bbe1707c7bcf03) It gives me pause to 'fix' a test by changing the assertion... (It's interesting that this passed for the TravisCI build at 2.0.0. release time - maybe a hapi change in the interim?).

I don't have much experience with submitting a pull request. Feel free to advise or let me know how to contribute more effectively.

### CODE REVIEW

**Comment by ratsimba** [comment]
Great

File: test/index.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ describe('Oppsy', () => {
                 total: 1,
                 disconnects: 1,
                 statusCodes: {
-                    '200': 1
+                    '499': 1
```
Comment (arb) [inline test/index.js]: Why this change?
